### ISSUE TITLE
Something went wrong with writing the config file. Please message Master Of WebM to figure it out.

### ISSUE THREAD

**Issue by azumukupoe** [MasterOfWebM/WebM-Converter#15]

**Comment by jmillemedia** [comment]
same problem. gives errormessage on startup and then the 'convert' button doesnt work. On my windows7 PC mow works with no error message. Only on win10 I get that.

**Comment by jmillemedia** [comment]
Stupid question, is it maybe possible to remove the font check? Are those only needed for subtitles? Many people don't use subtitles. But maybe this is a bigger issue because the start button is grayed out.

**Comment by krystiangorecki** [comment]
No chance. This repo is dead since years.

**Comment by jmillemedia** [comment]
That's actually fine. I just figured out that an older build works. It's amazing how things get worse with progress, it's everywhere in modern programming

**Comment by PLUHA33** [comment]
I had ffmpeg.exe in my System32 dir and I was able to fix this issue by adding a PATH variable to ffmpeg bin directory and then deleting ffmpeg.exe from System32

**Comment by atorralb** [comment]
true, modern programming is messing up the things that used to work... no wonder why I left development
